tests/core.js

```diff
@@ -375,16 +375,25 @@ require([
                 mug = util.addQuestion("Text", "text");
             });
 
-            _.each(["relevantAttr", "calculateAttr", "label"], function (attr) {
+            function testValidationError (attr, property) {
+                property = property || attr;
+                assert.deepEqual(mug.messages.get(attr), []);
```

Comment:
should this do `mug.messages.get(property)` instead? and the same for the last line in this function